CMakeLists.txt

```diff
@@ -59,23 +69,24 @@ add_subdirectory(include/beman/optional26)
 
 add_subdirectory(examples)
```

Comment:
**TODO**: examples should be optional with an `option(OPTIONAL26_ENABLE_EXAMLPES ...)`